internal/config/utils.go

```diff
@@ -1,14 +1,26 @@
 package config
 
 import (
+	"errors"
 	"fmt"
 	"os"
 	"reflect"
 
+	"github.com/joho/godotenv"
 	clowder "github.com/redhatinsights/app-common-go/pkg/api/v1"
 )
 
+var ErrMissingEnvTag = errors.New("missing 'env' tag in config field")
+var ErrUnsupportedFieldType = errors.New("unsupported config field type")
+
 func LoadConfigFromEnv(conf *ImageBuilderConfig) error {
+	if _, err := os.Stat("local.env"); err == nil {
```

Comment:
(nitpick). I would just `if err := godotenv.Load(); err != nil && !os.IsNotExist(err) { return err}` here instead of first stat/open because doing it in two steps is a TOCTOU issue. Not a big deal here of course but I would avoid it out of the general principle.

Reply:
I like your codesnipet, I've come up  with the same (similar), tested it and works fine :)